vmas/simulator/environment/gymnasium.py

```diff
@@ -0,0 +1,183 @@
+#  Copyright (c) 2022-2024.
+#  ProrokLab (https://www.proroklab.org/)
+#  All rights reserved.
+from typing import List, Optional
+
+import gym
+import gymnasium
+import numpy as np
+import torch
+
+from vmas.simulator.environment.environment import Environment
+from vmas.simulator.utils import extract_nested_with_index
+
+
+def _convert_space(space: gym.Space) -> gymnasium.Space:
```

Comment:
I really would like to avoid mainitaining this function, does gymnasium not have a conversion tool in their library?